plugin-name/composer.json

```diff
@@ -33,8 +34,7 @@
       "yahnis-elsts/plugin-update-checker": "^4.9"
    },
    "require-dev": {
-      "szepeviktor/phpstan-wordpress": "^0.6.0",
-      "szepeviktor/phpstan-phpdoc": "^0.2.1",
```

Comment:
Why did you removed that package? It is not now integrated in the other one?

Reply:
phpstan-phpdoc is an excerpt of PHPStan's documentation-related rules: no new code there.